rpcs3/rpcs3qt/tooltips.h

```diff
@@ -1,4 +1,4 @@
-#pragma once
+﻿#pragma once
```

Comment:
This file is kept in UTF-8 intentionally, don't change this.

Reply:
It only adds missing BOM I believe.

Reply:
Doesn't matter. Some string looks bad when you use BOM

Reply:
Which strings? Without BOM, VS may arbitrarily pick current locale, it doesn't seem to detect UTF-8 in all cases.

Reply:
Should be back to UTF-8 now